react-native-scripts/src/scripts/eject.js

```diff
@@ -191,11 +191,15 @@ from \`babel-preset-expo\` to \`babel-preset-react-native-stage-0/decorator-supp
       // missing native modules will cause
       delete pkgJson.dependencies.expo;
       delete pkgJson.devDependencies['react-native-scripts'];
+      delete pkgJson.devDependencies['jest-expo'];
 
       pkgJson.scripts.start = 'react-native start';
       pkgJson.scripts.ios = 'react-native run-ios';
       pkgJson.scripts.android = 'react-native run-android';
 
+      pkgJson.jest.preset = 'react-native';
```

Comment:
I think this is probably fine in 99.9% of the cases, but since we are here making changes, maybe we want to verify that the preset is set to "jest-expo" before we change it. if it is not, then we might want to back out of making the change and warn the user

Reply:
@brentvatne sounds good! What information would you add in this warning? I don't have any idea)

Reply:
`Warning: it looks like you've changed the Jest preset from jest-expo to ${pkjJson.jest.preset}. We recommend you make sure this Jest preset is compatible with ejected apps.`

Something like that?

Reply:
Looks good @anp!